docs/howto_wallet_desktop.md

```diff
@@ -0,0 +1,45 @@
+---
+id: howto_wallet_desktop
+title: Desktop
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Desktop Wallet
+
+There are multiple options for creating a Hydration account on a desktop, including browser extensions (like [Talisman](https://www.talisman.xyz/) and [Subwallet](https://www.subwallet.app/download.html) extensions) and native desktop wallets (like [Nova Spectre](https://novaspektr.io/)).
+
+This guide will walk you through setting up your account using the Talisman wallet.
+
+## Setting Up Talisman
+To get started, download the **Talisman extension** from the [official website](https://www.talisman.xyz/). The Talisman extension supports most modern browsers.
+
+## Creating Your Account
+1. Once you’ve downloaded and installed the extension, launch it and click on the **Add Account** button.
+2. Select **Polkadot** as the account, and input the account name of your choice. Then click **Create**.
+
+<div style={{textAlign: 'center'}}>
+  <img alt="Nova Wallet Landing Page" src={useBaseUrl('/howto_wallet_desktop/talisman_desktop.jpg')}
+  width="600px" />
+</div>
+
+3. Your **recovery phrase** will be displayed to you on the next page. Ensure you copy this down on a piece of paper and keep it safe.
+:::danger
+Do not share your mnemonic phrase with anyone. It’s advisable to not take screenshots of this phrase. Write it down on a sheet of paper, and keep it in a safe place.
+:::
+4. Once you’ve secured your recovery phrase, click on the **Verify Recovery Phrase** button
+5. Verify your phrase by inputting it in the correct order.
+6. To copy your Hydration address, click on the **Copy** button. Scroll down and look for Hydration, then click the copy icon to copy the address
+
+## Using Talisman
+<div style={{textAlign: 'center'}}>
+  <img alt="Nova Wallet Landing Page" src={useBaseUrl('/howto_wallet_desktop/talisman_copy_address.gif')}
+  width="600px" />
+</div>
+
+To view your balance, navigate to your account portfolio by clicking on the **Portfolio** button on the left sidebar.  Search for Hydration or HDX using the search bar.
```

Comment:
This is also not complete - please add the step of connecting Talisman to the Hydration app